packages/components/src/internal/query/api.ts

```diff
@@ -228,6 +234,20 @@ function applyColumnMetadata(schemaQuery: SchemaQuery, rawColumn: any): QueryCol
         if (columnMeta) {
             columnMeta = columnMeta.toJS();
         }
+        // special case for assay schema to allow for metadata to be applied to all protocols base tables
```

Comment:
@labkey-susanh @labkey-nicka @labkey-alan As part of the customize view work for the LKSM epic, I needed to apply metadata props from the app to all of the assay runs and results grids. We were only supporting explicit matches for the schema/query in this `applyColumnMetadata` function. I added this special case for the `assay` schema to allow for the metadata to be applied to all runs/results queries in that parent schema. Thoughts?

Susan mentioned that ideally we'd have a more generic wildcard like syntax that the app metadata files can use for this type of thing. I wasn't sure if I wanted to go that far just yet.